src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class Intake extends SubsystemBase {
+  /** Creates a new Intake. */
+  TalonFX TopMotor;
```

Comment:
this should be lowerCamelCase
```suggestion
  TalonFX topMotor;
```